addons/website/static/src/scss/website.scss

```diff
@@ -187,6 +187,19 @@ $-seen-urls: ();
     }
 }
 
+#oe_main_menu_navbar:not(.o_hidden) + #wrapwrap {
```

Comment:
Indeed :+1: , o_hidden is added on #oe_main_menu_navbar in edit mode..
Testing it, I found and other bug with modals in edit mode and full screen mode,  the position "right" of the modals were not updated to zéro instead of the width of the editor panel.